test/integration/discovery/consul.bats

```diff
@@ -59,6 +59,31 @@ function teardown() {
 	retry 5 1 discovery_check_swarm_info
 }
 
+@test "consul discovery: check for engines departure" {
```

Comment:
Could the name be consistent with `file`?
https://github.com/abronan/swarm/blob/add_ephemeral_support/test/integration/discovery/file.bats#L56

Either call this "node removal" or change file to "check for engines departure"